gunicorn/config.py

```diff
@@ -1856,14 +1873,28 @@ class SSLVersion(Setting):
     name = "ssl_version"
     section = "SSL"
     cli = ["--ssl-version"]
-    validator = validate_pos_int
+    validator = validate_ssl_version
     default = ssl.PROTOCOL_SSLv23
     desc = """\
-    SSL version to use (see stdlib ssl module's)
+    SSL version to use.
+============= ============
```

Comment:
Is there a reason not to indent the table?

```rst
    SSL version to use.

    ============= ============
    --ssl-version Description
    ============= ============
```

Reply:
And please add a newline after "SSL version to use." Some versions of docutils want a newline before a table.